modules/migration-mtc-release-notes-1-5.adoc

```diff
@@ -37,3 +39,12 @@ This release has the following known issues:
 * PV resizing does not work as expected for AWS gp2 storage unless the `pv_resizing_threshold` is 42% or greater. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1973148[*BZ#1973148*])
 * If a migration fails, the migration plan does not retain custom PV settings for quiesced pods. You must manually roll back the migration, delete the migration plan, and create a new migration plan with your PV settings. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1784899[*BZ#1784899*])
 * Microsoft Azure storage is unavailable if you create more than 400 migration plans. The `MigStorage` custom resource displays the following message: `The request is being throttled as the limit has been reached for operation type`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977226[*BZ#1977226*])
+
+[id="technical-changes-1-5_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+* You must manually install the legacy {mtc-full} Operator version 1.5.1 on {product-title} versions 3.7 to 4.5 because the custom resource definition API version has changed from `v1beta` to `v1`.
+* You must install the {mtc-full} Operator with the Operator Lifecycle Manager on {product-title} versions 4.6 to {product-version}.
+* You can migrate applications from a source cluster with {mtc-short} 1.5.1 to a target cluster with {mtc-short} 1.6.x as long as the {mtc-short} web console and the `MigrationController` pod are running on the target cluster.
```

Comment:
`as long as the {mtc-short} web console and the `MigrationController` pod are running on the target cluster.`, this needs to double confirm with Erik, my understanding is that no matter the `MigrationController` pod are running on the target cluster or source cluster, it should be able to migration applications from a source cluster to target cluster

Reply:
Will update after Erik confirms.